test/integration_test_helper.rb

```diff
@@ -20,18 +20,40 @@
 end
 
 Selenium::WebDriver::Chrome::Service.driver_path = ENV['TESTDRIVER_PATH'] || Foreman::Util.which('chromedriver', Rails.root.join('node_modules', '.bin'))
+
+options = Selenium::WebDriver::Chrome::Options.new
+options.args << '--disable-gpu'
+options.args << '--no-sandbox'
+options.args << '--window-size=1024,768'
+options.args << '--headless' unless ENV['DEBUG_JS_TEST'] == '1'
+
+selenium_remote_host = ENV.fetch('SELENIUM_REMOTE_HOST', nil)
+selenium_remote_port = ENV.fetch('SELENIUM_REMOTE_PORT', 4444)
```

Comment:
This can of course be done inside the registration. Currently the registration depends on selenium_remote_host.

Would you always register the `selenium_chrome_remote` or would you make it dependent on `JS_TEST_DRIVER`?

Reply:
I'd also always register it. Seems more straight forward